<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGame</class>
 <widget class="QDialog" name="NewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>189</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Player1: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Player2: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Player3: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Player4: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_4"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Player5: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_5"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Player6: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_6"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Player7: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_7"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Player8: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_8"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Player9: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_9"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Player10:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_pl_10"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
